lib/kotkowo_web/live/news_live/latest_news.ex

```diff
@@ -0,0 +1,25 @@
+defmodule KotkowoWeb.NewsLive.LatestNews do
+  @moduledoc false
+  use KotkowoWeb, :live_view
+
+  alias Kotkowo.GalleryImage
+  alias Kotkowo.StrapiClient
+
+  @impl true
+  def mount(_params, _session, socket) do
+    {:ok, header_news} = StrapiClient.list_announcements(1)
+    {:ok, popular_news} = StrapiClient.list_announcements(2, 1)
+    {:ok, news} = StrapiClient.list_announcements(4, 3)
+    {:ok, article} = StrapiClient.get_article(Integer.to_string(hd(header_news).id))
```

Comment:
What is this fetching? It looks like it's refetching. If it's fetching a related article rename it to `get_article_for_announcement(announcement_id)`

I have no idea how this passed the formatter check. 
Fix this.